src/components/common/Button/index.tsx

```diff
@@ -1,6 +1,38 @@
+import { PropsWithChildren } from 'react';
+import style from './style.module.scss';
+
 // TODO: Implement generic buttons that follow our design system
```

Comment:
Does the TODO comment still apply? Or is the Button component done?

Reply:
I don't think the comment applies anymore, I'll remove it